migrationmanagers/migrationmanagers.go

```diff
@@ -44,9 +44,9 @@ func NewWithDependencies(migration migrations.IMigration, server servers.IServer
 	}
 }
 
-// Save does the whole operation of validating and persisting a
+// CreateMigration does the whole operation of validating and persisting a
 // migration to disk, and updating the schema
-func (m *MigrationManager) Save() error {
+func (m *MigrationManager) CreateMigration() error {
```

Comment:
This is a straight rename. migrationmanagers.CreateMigration feels like the best description of what this does. You'd never use this to re-save a modified migration or anything, so Save felt wrong.